holoviews/util/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from collections import defaultdict
 from types import FunctionType
+from inspect import Parameter, Signature
```

Comment:
You'll have to inline the import for Python 2 tests to pass:

```
  File "/home/travis/build/holoviz/holoviews/holoviews/__init__.py", line 11, in <module>
    from . import util                                       # noqa (API import)
  File "/home/travis/build/holoviz/holoviews/holoviews/util/__init__.py", line 5, in <module>
    from inspect import Parameter, Signature
ImportError: cannot import name Parameter
```

Reply:
Thanks. Fixed in 0a808f9.